apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -1379,8 +1379,8 @@ describe('teacherSectionsRedux', () => {
       );
       server.respondWith(
         'GET',
-        '/dashboardapi/sections/valid_course_offerings',
-        successResponse([])
+        '/dashboardapi/sections/available_participant_types',
```

Comment:
Intentional: Realized there were two calls to valid course offerings and none to available participant types in this test